app/controllers/api/v1/ability.rb

```diff
@@ -302,6 +302,7 @@ def package_abilities
           can %i[show create update destroy], PackageSet
           can %i[index], Restriction
           can %i[index], PackagesInventory
+          can %i[index], UserFavourite
```

Comment:
That's not right, a user should only be able to see his/her favourites

```suggestion
can [:index], UserFavourite,   user_id: @user_id
```